pkg/mcs/scheduling/server/rule/watcher.go

```diff
@@ -62,6 +63,17 @@ type Watcher struct {
 	ruleWatcher  *etcdutil.LoopWatcher
 	groupWatcher *etcdutil.LoopWatcher
```

Comment:
I still worry about if two watchers will get the event disordered and overwrite the same rule.

Reply:
I will add a test about it. If necessary, I will merge these watchers.